README.md

```diff
@@ -117,10 +115,16 @@ shell with:
 If you're running inside of an msys shell, however, you can run:
 
 ```sh
-$ ./configure --build=x86_64-pc-windows-msvc
-$ make && make install
+$ ./x.py build --build=x86_64-pc-windows-msvc
+$ ./x.py dist --build=x86_64-pc-windows-msvc --install
```

Comment:
@retep998 
>commands are identical no matter what shell you're in

Default build triples are different.
That's why you need to explicitly specify `x86_64-pc-windopws-msvc` in msys/mingw and `x86_64-pc-windopws-gnu` in powershell/cmd.
This is probably the only notable difference in commandlines, but it needs to be mentioned.
`--build=x86_64-pc-windopws-msvc` looks like a correct invocation, but I'm not sure (I use `config.toml`).